packages/runtime-dom/src/components/CSSTransition.ts

```diff
@@ -262,22 +265,38 @@ function getTransitionInfo(
   }
 }
 
+function getMaximumNumber<T>(
+  array: T[],
+  valueCallback: (value: T, index: number) => number,
+  intialMaximumNumber = -Infinity
+) {
+  let maximum = intialMaximumNumber
+
+  for (let i = 0; i < array.length; i++) {
+    const current = valueCallback(array[i], i)
+
+    if (current > maximum) {
+      maximum = current
+    }
+  }
+
+  return maximum
+}
+
 function getTimeout(delays: string[], durations: string[]): number {
   while (delays.length < durations.length) {
     delays = delays.concat(delays)
   }
-  return Math.max.apply(
-    null,
-    durations.map((d, i) => {
-      return toMs(d) + toMs(delays[i])
-    })
+  return getMaximumNumber(
+    durations,
+    (duration, i) => toMs(duration) + toMs(delays[i])
   )
 }
 
 // Old versions of Chromium (below 61.0.3163.100) formats floating pointer
 // numbers in a locale-dependent way, using a comma instead of a dot.
 // If comma is not replaced with a dot, the input will be rounded down
 // (i.e. acting as a floor function) causing unexpected behaviors
-function toMs(s: string): number {
+function toMs(s: string) {
```

Comment:
Here the return value serves documentation purposes.
```suggestion
function toMs(s: string): number {
```